samples/infiniteStreaming.js

```diff
@@ -271,7 +271,7 @@ require(`yargs`)
     },
     streamingLimit: {
       alias: 's',
-      default: 10000,
+      default: 290000,
```

Comment:
Just wondering, why not 300000 for 5 minutes even?

Reply:
+1 was wondering the same thing

Reply:
If we wait for the full 5 minutes, then the API will return an error that we've reached the limit, so we cut it off a few seconds early and then restart the stream with the buffered audio to provide a sort of "infinite" streaming.

Reply:
That would have been a cool comment to leave in the code :)